### PR TITLE
Ride cpp

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: src/openrct2/ride/Ride.cpp [hunk 1]
```diff
@@ -8309,7 +8969,8 @@ void fix_ride_entrance_and_exit_locations()
             {
                 tileElement = map_get_ride_exit_element_at(exitLoc.x * 32, exitLoc.y * 32, entranceExitHeight, false);
 
-                if (tileElement == NULL || tileElement->properties.entrance.ride_index != rideIndex || tile_element_get_station(tileElement) != stationIndex)
+                if (tileElement == NULL || tileElement->properties.entrance.ride_index != rideIndex ||
```
Comment (Gymnasiast) [inline src/openrct2/ride/Ride.cpp]: Shouldn't this be `== nullptr`?
Also, I'd prefer if this if statement was either vertical or horizontal.

File: src/openrct2/ride/Ride.cpp [hunk 2]
```diff
@@ -8131,17 +8788,14 @@ uint8 ride_entry_get_first_non_null_ride_type(rct_ride_entry * rideEntry)
...
 
 bool ride_type_supports_boosters(uint8 rideType)
 {
-    if (rideType == RIDE_TYPE_LOOPING_ROLLER_COASTER ||
-        rideType == RIDE_TYPE_CORKSCREW_ROLLER_COASTER ||
-        rideType == RIDE_TYPE_TWISTER_ROLLER_COASTER ||
-        rideType == RIDE_TYPE_VERTICAL_DROP_ROLLER_COASTER ||
-        rideType == RIDE_TYPE_GIGA_COASTER ||
-        rideType == RIDE_TYPE_JUNIOR_ROLLER_COASTER)
+    if (rideType == RIDE_TYPE_LOOPING_ROLLER_COASTER || rideType == RIDE_TYPE_CORKSCREW_ROLLER_COASTER ||
```
Comment (Gymnasiast) [inline src/openrct2/ride/Ride.cpp]: This clang autoformatting makes stuff harder to read. Can it not be configured to make the if statement vertical if it's too long?

**Comment by IntelOrca** [comment]
@janisozaur can you rebase this?

**Comment by Gymnasiast** [comment]
Superseded by #6899.

**Pull request closed without merging**